### PR TITLE
Creates html file for Nelson Mosquera

### PR DESCRIPTION
**Pull request by monotera**
Includes a new html file in folder git

### CODE REVIEW

File: git/Nelson_Mosquera.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Nelson Mosquera</title>
+  </head>
+  <body>
+    <p>Git changes Nelson Mosquera</p>
+  </body>
+</html>
```
Comment (fabianmarinog) [inline git/Nelson_Mosquera.html]: remove this empty space at the end of the file.

Comment (monotera) [reply git/Nelson_Mosquera.html]: In my machine the space at the end does not appear.

File: git/Nelson_Mosquera.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Nelson Mosquera</title>
+  </head>
+  <body>
+    <p>Git changes Nelson Mosquera</p>
```
Comment (fabianmarinog) [inline git/Nelson_Mosquera.html]: This should be a heading (`<h1>`)

Comment (monotera) [reply git/Nelson_Mosquera.html]: I have made the required changes

**Pull request merged**